packages/cli/src/Db.ts

```diff
@@ -104,10 +104,25 @@ export async function init(): Promise<IDatabaseCollections> {
 			throw new Error(`The database "${dbType}" is currently not supported!`);
 	}
 
+	let loggingOption: LoggerOptions = (await GenericHelpers.getConfigValue(
+		'database.logging.enabled',
+	)) as boolean;
+
+	if (loggingOption) {
+		const optionsString = (await GenericHelpers.getConfigValue(
+			'database.logging.options',
+		)) as string;
+
+		loggingOption = optionsString.split(',') as LoggerOptions;
```

Comment:
I am not sure how it behaves, but perhaps we should `trim()` after splitting just to be on the safe side.